Libraries/Settings/NativeSettingsManager.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+export interface Spec extends TurboModule {
+  +getConstants: () => {|
+    settings: Object,
```

Comment:
You should avoid using Object

`Object` is `any` since 0.88

```js
settings: { +[key: string]: string },
```

Reply:
Just as an FYI- Object is how it was recommended to do it for the exercise- I did update to your suggestion however.